### ISSUE TITLE
Why are there contracts with more than 1 contracting authority?

### ISSUE THREAD

**Issue by jindrichmynarz** [opendatacz/VVZ_extractor#3]
There are public contracts, which list more than 1 entity as `pc:contractingAuthority`. If you look at these contracts, the contracting authorities they list are likely identical, but the extractor creates a superfluous URIs for them. For example, see the [results of this query](http://goo.gl/pPxgkq):

```
PREFIX gr: <http://purl.org/goodrelations/v1#>
PREFIX pc: <http://purl.org/procurement/public-contracts#>

SELECT ?s
(COUNT(DISTINCT ?contractingAuthority) AS ?count)
(GROUP_CONCAT(?legalName; separator = ", ") AS ?legalNames)
WHERE {
  GRAPH <http://linked.opendata.cz/resource/dataset/vestnikverejnychzakazek.cz> {
    ?s pc:contractingAuthority ?contractingAuthority .
    OPTIONAL {
      ?contractingAuthority gr:legalName ?legalName .
    }
  }
}
GROUP BY ?s
HAVING (COUNT(DISTINCT ?contractingAuthority) > 1)
ORDER BY DESC(?count)
```

Since `pc:contractingAuthority` is [used only once](https://github.com/opendatacz/VVZ_extractor/blob/master/xslt/pc.xsl#L138) in the extractor's XSLT, it seems it might be instead a problem of assigning multiple contracts with the same URI.

**Comment by michalpomykacz** [comment]
I think it may be caused by the VVZ Extractor processed (via SOAP API) some of the same contracts which are present in VVZ dump (processed separately), provided by NESS. 
URI of each ContractingAuthority contains UUID, while contract URI is still the same.

Solution could be delete leftover ContractingAuthorities. That issue should no longer appear.

**Comment by jindrichmynarz** [comment]
How could you prevent this from happening again, so that the incremental pipeline doesn't extract a contract that has been already extracted to RDF?

**Comment by michalpomykacz** [comment]
Yes, it shouldn´t be. Pipeline stores XMLs of extracted contracts in cache. Extractor doesn´t process the contracts that has already cached. Problem was VVZ dump (of XMLs) isn´t part of the cache and part of the contracts from dump reaches 2014 year, since when extractor running.
I previously thought that the dump contains contracts only until the end of 2013
